vue.config.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
   lintOnSave: false,
+
+  pluginOptions: {
+    lintStyleOnBuild: false,
+    stylelint: {},
```

Comment:
Aren't both of these the defaults? I'd vote to remove them if so.

Reply:
Yes, we can remove these. (But seems the pluginOptions needs to stay, or the @ascendancyy/vue-cli-plugin-stylelint will throw exception)